tests/processes/LogPrinterThreadTest.py

```diff
@@ -2,14 +2,10 @@
 import unittest
 
 from coala_utils.ContextManagers import retrieve_stdout
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.processes.LogPrinterThread import LogPrinterThread
 
 
-class TestPrinter(LogPrinter):
-
-    def __init__(self):
-        LogPrinter.__init__(self, self)
+class TestPrinter():
```

Comment:
Yeah, we do.
I've added it to cover the line https://github.com/coala/coala/pull/4320/files/c367b4ca2e3157746a51ed3ad13abdbfeabb21ec#diff-16aa1ce38a1d20ae5cc6c79934589eb2R45.
May be I could have done it by calling `LogPrinterMixin().log_message(...)`, but this seemed a bit cleaner way.